xml_handlers/OpenCoverTar_handler.php

```diff
@@ -231,29 +231,28 @@ public function readSourceFile($buildid, $fileinfo) {
        $inlongComment= false;
        if ($this->coverageFile) {
            foreach($fileContents as $key=>$line) {
+              $trimmedLine = trim($line);
+              $displayLine = rtrim($line);
               // Matches the beginning of a comment block
-              if(preg_match("/\/[*]+/", trim($line))) {
+              if(preg_match("/\/[*]+/", $trimmedLine)) {
                   $inlongComment=true;
               }
 
-              $this->coverageFile->File .= rtrim($line).'<br>';
-              if((preg_match("/^\/\//", trim($line))) or
-                (preg_match("/using /", trim($line))) or
-                (preg_match("/^namespace/", trim($line))) or
-                (preg_match("/^public/", trim($line))) or
-                (preg_match("/^protected/", trim($line))) or
-                (preg_match("/^private/", trim($line))) or
-                (preg_match("/^\[/", trim($line))) or
-                (preg_match("/[{}]/", trim($line))) or
-                ("" == trim($line)) or
-                ($inlongComment)) {
-                // ignore these lines
-              } else {
-                  // Assumed to be executable code
+              $this->coverageFile->File .= $displayLine.'<br>';
+              if(!((preg_match("/^\/\//", $trimmedLine)) or
```

Comment:
There is a new set of parentheses there which make it negate the whole conditional not just the first one entry.  It just doesn't quite show in the diff.  I can redo the block to make that a little clearer if you want